tests/bootstrap.php

```diff
@@ -5,6 +5,7 @@
  */
 
 require_once dirname( __DIR__ ) . '/enpii-base-bootstrap.php';
+require_once dirname( __DIR__ ) . '/vendor/autoload.php';
```

Comment:
The `enpii-base-bootstrap.php` already include the autoload.php, you need to review if we have the condition to avoid to load the autoload again